src/extensions/surveys/components/QuestionTypes.tsx

```diff
@@ -81,20 +81,28 @@ export function OpenTextQuestion({
 }: CommonQuestionProps & {
     question: BasicSurveyQuestion
 }) {
+    const inputRef = useRef<HTMLTextAreaElement>(null)
     const [text, setText] = useState<string>(() => {
         if (isString(initialValue)) {
             return initialValue
         }
         return ''
     })
 
+    useEffect(() => {
+        setTimeout(() => {
+            inputRef.current?.focus()
+        }, 100)
+    }, [])
```

Comment:
logic to autofocus on the open question input moved to here, where it makes more sense.